operations/_objects.md

```diff
@@ -182,6 +182,15 @@ A [Dictionary](#dictionary) object where the keys are [Language](languages.md#la
 | `NetValue` | number | required | The net value that the tax is calculated from. |
 | `TaxValue` | number | required | The value of the tax. |
 
+### Hybrid identifier
+
+Hybrid identifier is a string value with specific format that allows providing other unique identifier than just unique identifier of the entity. Using other identifiers require providing corresponding prefix as part of the value.
```

Comment:
For docs annotations I often find ChatGPT version sounding better:
```suggestion
A hybrid identifier is a string with a specific format that allows the use of alternative unique identifiers, in addition to the entity's unique identifier. When using these alternative identifiers, a corresponding prefix must be included as part of the value.
```

Reply:
I trier Copilot but that wasn't helpful much 🤣